fetch.py

```diff
@@ -23,6 +28,7 @@ def getData(dep):
 
 	courses = []
 
+	print(response.text)
```

Comment:
@debajyotidasgupta Do you think we should remove this?

Reply:
Yeah okay, I will